src/internal/clusterstate/v2.8.0_test.go

```diff
@@ -14,6 +17,15 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/testetcd"
 )
 
+func sortCommits(commitInfos []v2_8_0.CommitInfo) {
+	sort.Slice(commitInfos, func(i, j int) bool {
+		if commitInfos[i].CommitID == commitInfos[j].CommitID {
```

Comment:
This is possible??

Reply:
for our test data, it is. I assume it sorts the commit Ids lexicographically